lib/_internal/file_picker_web.dart

```diff
@@ -185,6 +186,47 @@ class FilePickerWeb extends FilePicker {
     return files == null ? null : FilePickerResult(files);
   }
 
+  @override
+  Future<String?> saveFile({
+    String? dialogTitle,
+    String? fileName,
+    String? initialDirectory,
+    FileType type = FileType.any,
+    List<String>? allowedExtensions,
+    Uint8List? bytes,
+    bool lockParentWindow = false,
+  }) async {
+    if (bytes == null) {
```

Comment:
Just to be sure we do not start an empty download.

```suggestion
    if (bytes == null || bytes.isEmpty) {
```